### DISCUSSION TITLE
Due to PHP limitations, the fork driver may not be used within web requests.

### DISCUSSION THREAD

**Discussion by LfJohnVo** [laravel/framework#53430] [Q&A]
I have this issue on a simple concurrency code test: 

`
        [$first, $second] = Concurrency::run([
            fn () => DB::table('users')->count(),
            fn () => Empleado::exists()
        ]);

        dd($first, $second);
`

i dont know what to do :(

**Comment by gazzoy** [comment]
What driver do you use?

[As documented](https://laravel.com/docs/11.x/concurrency#how-it-works), the `fork` driver would ONLY works in php-cli context:
